### PR TITLE
Use workflow merging feature

### PR DESCRIPTION
**Pull request by knopers8**

### CODE REVIEW

File: Framework/src/runQC.cxx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file    runQC.cxx
+/// \author  Piotr Konopka
+///
+
```
Comment (Barthelemy) [inline Framework/src/runQC.cxx]: Could you add a few lines to explain what it is ? like you did for runDataProducer

**Comment by Barthelemy** [comment]
This is an excellent development !
I will test it locally before putting the approval.

**Comment by Barthelemy** [comment]
Tested, works very well, instructions clear and working. Congrats !

**Comment by knopers8** [comment]
It looks like there is a bug in the CI - it is being shown that the PR is WIP. When entering into the details, it actually refers to another PR (The title "[WIP] Reduce compilation time of the QC project" contains "WIP".)

File: Framework/src/runQC.cxx (hunk 1 above)
Comment (knopers8) [reply Framework/src/runQC.cxx]: will do

**Pull request merged**